content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -293,30 +293,29 @@ Containers within the Pod see the system hostname as being the same as the confi
 `name` for the Pod. There's more about this in the [networking](/docs/concepts/cluster-administration/networking/)
 section.
 
-## Privileged mode for containers
```

Comment:
I like this refactoring, i.e. consolidating security-related topics into a single section.
What I'd suggest is to leave some pointers here so that users know there are security
supports and where to find them.

Reply:
Ive currently put in links to the securitycontext how-to page and the new page we're adding. Maybe also a pointer to the security overview and security checklist?